shared-log4j/pom.xml

```diff
@@ -1,18 +1,18 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.opentestsystem.shared</groupId>
-    <artifactId>shared-master</artifactId>
-    <version>1.0.1-BUILD-SNAPSHOT</version>
-  </parent>
-  <artifactId>shared-log4j</artifactId>
-  <name>Log4j utilities</name>
-  <description>Utilities to extend Log4j logging framework</description>
-  <dependencies>
-	<dependency>
-		<groupId> log4j</groupId>
-		<artifactId>log4j</artifactId>
-		<scope>provided</scope>
-	</dependency>
-  </dependencies>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">

```

Comment:
Not sure if this is intentional, but the main change here looks to be going from `SNAPSHOT` to none snapshot.  However, it seems the entire file was formatted which is why we're seeing full red/green on the change.  We've been avoiding reformatting files on other changes to legacy.  Not sure if it matters here.